explorer/app/viewModelBuilders/catalog/altos-labs-catalog/common/viewModelBuilders.ts

```diff
@@ -155,6 +148,40 @@ export const buildInitiative = (
   };
 };
 
+/**
+ * Build props for S3 URI cell component from the given Altos Labs entity.
+ * @param altosLabsCatalogFile - Altos Labs catalog file entity.
+ * @returns Model to be used as props for the S3 URI cell.
+ */
+export const buildS3Uri = (
+  altosLabsCatalogFile: AltosLabsCatalogFile
+): React.ComponentProps<typeof C.Link> => {
+  try {
+    const url = new URL(altosLabsCatalogFile.s3Uri);
+    const prefix = url.pathname.match(/\/public.+/g)?.shift();
```

Comment:
Can we create a reusable function here that can go into a common ViewModel builder  area that takes in a S3URI and returns the AWS Console URL like buildAWSConsoleURL(S3URI)? Thanks!